command/config/write/config_write_test.go

```diff
@@ -470,24 +732,74 @@ func TestParseConfigEntry(t *testing.T) {
 				Name = "main"
 				Splits = [
 				  {
-					Weight        = 99.1
+					Weight        = 97.1
 					ServiceSubset = "v1"
 				  },
+				  {
+					Weight        = 2,
```

Comment:
This was just to verify `float64` vs `int` didn't do something weird outside of those generic `config` blocks.